libr/bin/mangling/cxx.c

```diff
@@ -1,6 +1,7 @@
 /* radare - LGPL - Copyright 2013-2018 - pancake */
 
 #include <r_bin.h>
+#include <bin_private.h>
```

Comment:
We can use ./ like the line below and remove the -I.

Reply:
Yes we could. I did this just to avoid having different "../../" in different parts of the libr/bin codebase (e.g. sometimes it's used in libr/bin/bin.c, others in libr/bin/d/, etc.)

Reply:
do that then